subscription-manager.spec

```diff
@@ -1251,6 +1251,7 @@ find %{buildroot} -name \*.py* -exec touch -r %{SOURCE0} '{}' \;
 
 %files -n python3-cloud-what
 %defattr(-,root,root,-)
+%attr(755,root,root) %dir %{_var}/cache/cloud-what
```

Comment:
I guess the mode here ought to be 750 to match what the Makefile sets? And most likely we do not want non-root users to look into this directory.

Reply:
Is it really necessary? Maybe we would like to share here something that could be use by other non-root users in the future. If we will want restrict access to some cache files, then we can restrict this access directly on cache files. Thoughts?

Reply:
First of all, IMHO it makes sense for the mode set by the Makefile and the one used for packaging to be the same. Right now the former is 750, and the latter is 755.

Considering cloud-what is root-only at the moment, it makes sense to go with that assumption for other things as well. It is easier to loosen permissions later on, in case you want to switch away from root-only. It is much harder to do the opposite, i.e. tighten loose permissions, because you might miss something that relies on access to those resources.

Reply:
@ptoscano fixed.

Reply:
Thanks!